crates/cargo-lambda-build/src/compiler/mod.rs

```diff
@@ -7,7 +7,7 @@ use std::process::Command;
 mod cargo;
 use cargo::Cargo;
 mod cargo_zigbuild;
-use cargo_zigbuild::CargoZigbuild;
+use self::cargo_zigbuild::CargoZigbuild;
```

Comment:
<img width="1182" alt="Capture d’écran 2024-04-26 à 16 55 46" src="https://github.com/cargo-lambda/cargo-lambda/assets/2571084/0722c663-e8c9-45c3-81c4-a846614040cd">

Reply:
well... I don't have the issue anymore since I built the project using Rust `1.77.2` instead of `1.70.0`... so I removed this change ✅